main.go

```diff
@@ -74,15 +74,37 @@ func handleIssueComment(body []byte, gitRepos git.Repos, pullRequests PullReques
 	if !issueComment.IsPullRequest {
 		return SuccessResponse{"Not a PR. Ignoring."}
 	}
-	switch {
-	case isSquashCommand(issueComment.Comment):
+	commentCategory := parseComment(issueComment.Comment)
+	if commentCategory == regularComment {
+		return SuccessResponse{"Not a command I understand. Ignoring."}
+	}
+	if isAuthorized, err := isCollaborator(issueComment.Repository, issueComment.User, repositories); err != nil {
```

Comment:
Push most of this auth handling code to a helper function. The content is distracting in the context of this function.